addons/refuel/CfgVehicles.hpp

```diff
@@ -318,13 +311,20 @@ class CfgVehicles {
         GVAR(fuelCapacity) = 80;
     };
 
-    class Van_01_fuel_base_F: Van_01_base_F {
+    class Van_01_fuel_base_F: Van_01_base_F {};
+    class C_Van_01_fuel_F: Van_01_fuel_base_F {
         transportFuel = 0; //1k
         MACRO_REFUEL_ACTIONS
         GVAR(hooks)[] = {{0.38,-3.17,-.7},{-0.41,-3.17,-.7}};
         GVAR(fuelCargo) = 2000;
     };
-
+    class I_G_Van_01_fuel_F: Van_01_fuel_base_F {
+        transportFuel = 0; //1k
+        MACRO_REFUEL_ACTIONS
+        GVAR(hooks)[] = {{0.38,-3.17,-.7},{-0.41,-3.17,-.7}};
+        GVAR(fuelCargo) = 2000;
+    };
```

Comment:
jonpas, everything that inherits from `Van_01_fuel_base_F` is a fuel truck.
Only `C_Van_01_fuel_F` and `I_G_Van_01_fuel_F` do.
If BI adds the `transportFuel` properties to each one instead of the common base class, then we have to adjust. That does not mean that we have to imitate their strange practices.